src/main/java/DAO/TicketDAO.java

```diff
@@ -4,18 +4,18 @@
 import config.ConnectionConfiguration;
 import model.Ticket;
 import model.enums.TicketType;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
 import view.Printable;
 
-import java.sql.Connection;
-import java.sql.Date;
-import java.sql.PreparedStatement;
-import java.sql.ResultSet;
-import java.sql.SQLException;
+import javax.sql.DataSource;
+import java.sql.*;
 import java.time.LocalDate;
 import java.util.ArrayList;
 import java.util.Optional;
 import java.util.UUID;
 
+@Component
```

Comment:
Use `@Repository` for dao layers

Reply:
Done. I updated DAO layers with annotation @Repository instead of @Component